runtime/doc/eval.txt

```diff
@@ -7052,6 +7073,26 @@ sqrt({expr})						*sqrt()*
 		"nan" may be different, it depends on system libraries.
 
 
+stdioopen({opts})			 *stdioopen()*
+		In a nvim launched with the |--headless| option, this opens
+		stdin and stdout as a |channel|. This function can only be
+		invoked once per instance. See |channels-stdio| for more
+		information and examples. See also |v:stderrchan|
+
+		Returns a |channel| ID. Close the stdio descriptors with |chanclose()|.
+		Use |chansend()| to send data stdout, and
+		|rpcrequest()| and |rpcnotify()| to communicate with a RPC
+		soceket.
+
+		{opts} is a dictionary with these keys:
+		  |on_stdio|: callback invoked when stdin is written to.
+		  rpc      : If set, |msgpack-rpc| will be used to communicate
+			     over stdio
+		Returns:
+		  - The channel ID on success (this is always 1)
```

Comment:
Why? the function already returns the value. And one-time callable function signals that the channel is "owned by" the one that calls it, stdout is not expected to be used randomly by random plugins (but a fixed number is good for the external embedder's point of view) . But OTOH `v:stderr` tells that the channel "belongs" to nvim itself, and could be used by more than one (but it should still be used sparingly).